packages/grid/backend/grid/core/node.py

```diff
@@ -26,7 +26,7 @@
 mongo_store_config = MongoStoreConfig(client_config=mongo_client_config)
 
 
-client_config = SQLiteStoreClientConfig(path="/storage/")
```

Comment:
@madhavajay is this supposed to be `BACKEND_STORAGE_PATH`? If yes then I need to make the fixes across the compose files as well.

Reply:
I figured that `BACKEND_STORAGE_PATH` is not being used in syft. It's just used in hagrid to tweak the host volume path for credentials. I'll tweak the name of this to be more indicative of this.

Reply:
Additionally, these `/storage` also need to be made rootless. I'll make them `$HOME/data`